### PR TITLE
[h264e] Fix LA support on TGL

### PR DESCRIPTION
**Pull request by AntonGrishin**
LA TGL kernel already enabled, but query wasnt update.

### CODE REVIEW

File: _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp [hunk 1]
```diff
@@ -2482,7 +2482,7 @@ mfxStatus MfxHwH264Encode::CheckVideoParamQueryLike(
     }
 
     bool laEnabled = true;
-    if (platform > MFX_HW_ICL_LP)
+    if (platform > MFX_HW_ICL_LP && platform != MFX_HW_TGL_LP)
```
Comment (dmitryermilov) [inline _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp]: Please add a comment why we need such condition

Comment (dmitryermilov) [reply _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp]: Okay, it's VME dependency. Please name this accordingly. Please also check HEVC part. The shall the same condition there.

File: _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp [hunk 2]
```diff
@@ -2482,7 +2487,7 @@ mfxStatus MfxHwH264Encode::CheckVideoParamQueryLike(
     }
 
     bool laEnabled = true;
-    if (platform > MFX_HW_ICL_LP)
+    if (!isSupportVME(platform))
```
Comment (dmitryermilov) [inline _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp]: Two questions: 
# What about TGL?
# what about HEVC?

File: _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp [hunk 3]
```diff
@@ -244,6 +244,11 @@ namespace
             ;
     }
 
+    bool isSupportVME(eMFXHWType platform)
```
Comment (dmitryermilov) [inline _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp]: I'd prefer HasVMESupport()

File: _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp (hunk 2 above)
Comment (AntonGrishin) [reply _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp]: 1. <= TGL, so supports.
2. Didnt find any similar logic

File: _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp (hunk 3 above)
Comment (AntonGrishin) [reply _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp]: changed

**Pull request merged**